packages/cli/src/utils/get-config.ts

```diff
@@ -685,6 +685,133 @@ export async function promptAndStoreAnthropicKey(envFilePath: string): Promise<s
   }
 }
 
+/**
+ * Validates an Ollama API endpoint format
+ * @param endpoint The endpoint URL to validate
+ * @returns True if the endpoint appears valid
+ */
+export function isValidOllamaEndpoint(endpoint: string): boolean {
+  if (!endpoint || typeof endpoint !== 'string') return false;
+
+  try {
+    const url = new URL(endpoint);
+    return url.protocol === 'http:' || url.protocol === 'https:';
+  } catch {
+    return false;
+  }
+}
+
+/**
+ * Stores Ollama configuration in the .env file
+ * @param config The Ollama configuration to store
+ * @param envFilePath Path to the .env file
+ */
+export async function storeOllamaConfig(
+  config: { endpoint: string; model: string },
+  envFilePath: string
+): Promise<void> {
+  if (!config.endpoint || !config.model) return;
+
+  try {
+    // Read existing content first to avoid duplicates
+    let content = '';
+    if (existsSync(envFilePath)) {
+      content = await fs.readFile(envFilePath, 'utf8');
+    }
+
+    // Remove existing Ollama lines if present
+    const lines = content
+      .split('\n')
+      .filter(
+        (line) =>
+          !line.startsWith('OLLAMA_API_ENDPOINT=') &&
+          !line.startsWith('OLLAMA_MODEL=') &&
+          !line.startsWith('USE_OLLAMA_TEXT_MODELS=')
+      );
+
+    // Add new Ollama configuration
+    lines.push(`OLLAMA_API_ENDPOINT=${config.endpoint}`);
+    lines.push(`OLLAMA_MODEL=${config.model}`);
+    lines.push('USE_OLLAMA_TEXT_MODELS=true');
```

Comment:
we could make some `updateEnv` re-usable config for this, used many places, I did make a `mergeEnvs` function but new task later